lib/mocha.js

```diff
@@ -677,6 +677,20 @@ Mocha.prototype.forbidPending = function() {
   return this;
 };
 
+/**
+ * Mocha version as specified by "package.json".
+ *
+ * @name Mocha#version
+ * @type string
+ * @readonly
+ */
+Object.defineProperty(Mocha.prototype, 'version', {
+  value: require('../package').version,
+  configurable: false,
```

Comment:
FWIW these are the default values for `configurable` and `writable` and could be omitted.

Is there a reasoning behind `enumerable: true`?

Reply:
*ping* @plroebuck

Reply:
Defaults were explicit to imply intent (as there were only three boolean properties) rather than oversight. Found various examples (like [this](https://github.com/nodejs/node/blob/e7f710c1d4729890806610e5450d7a0edfb4ac11/lib/perf_hooks.js#L224)) within Node.js itself that did same.
`enumerable: true` allowed it to show up in `for` loop over properties.

You feel otherwise concerning the style?

Reply:
@ORESoftware, don't know how to interpret your "thumbs up"...